### PR TITLE
Translating the technical test to english

### PR DESCRIPTION
**Pull request by quentinus95**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,35 +75,34 @@ Ces identifiants sont également disponibles en base de données, dans le schém
...
-L'API `listingsapi` renvoie des pages de 20 annonces. Il vous faudra donc parcourir toutes les pages pour tous les arrondissements, via la paramètre `?page=<numero_de_page>`.
-
-Exemple: http://listingapi:5000/listings/32682?page=7
+The `listingapi` sends pages of 20 listings. You will have to browse all pages for all arrondissements, using the parameter `?page=<page_number>`.
 
-Le nombre de pages de résultat est différent pour chaque arrondissement : de zéro à plusieurs dizaines de pages. Il faudra imaginer un mécanisme s’adaptant au nombre de pages à parcourir. Il y a plusieurs manières de faire.
+Example: http://listingapi:5000/listings/32682?page=7
 
-### 1.3 Extraction des caractéristiques des annonces
+The number of pages is different for each arrondissement: from zero to dozens of pages. You will have to imagine a mechanism that can be adapted to the number of pages to browse. There are multiple implementations possible.
```
Comment (Yoan-MA) [inline README.md:82]:
```suggestion
The number of pages is different for each district: from zero to dozens of pages. You will have to imagine a mechanism that can be adapted to the number of pages to browse. There are multiple implementations possible.
```

File: README.md [hunk 2]
```diff
@@ -75,35 +75,34 @@ Ces identifiants sont également disponibles en base de données, dans le schém
...
-- `place_id` : identifiant de l’arrondissement (celui passé en paramètre de la recherche)
-- `price` : prix de mise en vente, en valeur entière d’euros
-- `area` : superficie du bien, en valeur entière de mètres carrés
-- `room_count` : nombre de pièces du bien, en valeur entière également ;
+### 1.3 Extracting listings properties
 
-Il se peut que les caractéristiques ne soient pas exposées comme souhaité par l'API `listingsapi`, mais que certaines d'entre elles soient à extraire. Attention aux appartements de 1 pièce qui sont notés « Studio »
+For each listing, we are interested in the following properties:
+- `listing_id`: listing identifier for MeilleursAgents;
+- `place_id`: arrondissement identifier (as passed in the query parameters);
```
Comment (Yoan-MA) [inline README.md:88]:
```suggestion
- `place_id`: district identifier (as passed in the query parameters);
```

File: README.md [hunk 3]
```diff
@@ -112,46 +111,43 @@ Voici les informations requises pour se connecter au serveur de base de données
...
+As for part 1, the web application is already working, but **can be improved**.
 
-Pour chaque arrondissement, le prix moyen par mètre carré réel est calculé pour être ensuite affiché sur le site web.
+### 2.1 - Displaying prices for each arrondissement
 
-Le code est dans l'endpoint `/geoms` dans `pricemap/pricemap/blueprints/api.py`. Comme pour la première partie, le code est fonctionnel mais peut être amélioré : vérification du calcul effectué, requête SQL, modélisation du stockage de la donnée, etc.
+When the page loads, the JavaScript code that generates the map calls the Python web application to get the list of geographical entities to display. The web application responds with GeoJSON formatted data that contains a list of arrondissements to display, their geometrical shape as well as an average price. The color of the shape depends on the price of the arrondissement that is represented, following the same color scale as the one currently used on MeilleursAgents website for Paris map.
 
-### 2.2 - Afficher des statistiques par arrondissement
+For each arrondissement, the average price per real square meter is computed, to be displayed on the website.
```
Comment (Yoan-MA) [inline README.md:124]:
```suggestion
For each district, the average price per square meter is computed, to be displayed on the website.
```

File: README.md [hunk 4]
```diff
@@ -1,53 +1,53 @@
-# Projet - Carte des prix à Paris
+# Backend technical test: Price map in Paris
 
-L’objectif est de réaliser une carte des prix moyens (€/m2) de mise en vente par arrondissement à Paris, accompagnée de statistiques, à partir d'une API d'annonces de Meilleurs Agents.
-Voici un exemple de ce qu’on veut obtenir au final :
+The objective is to build a price map of the average selling prices (€/m²) per arrondissment in Paris, with statistics, from an API of listings from MeilleursAgents.
```
[the remark this answers is not in the document]
Comment (quentinus95) [reply README.md:3]: I would go for MeilleursAgents and go for a global MA-to-AVIV change later on

**Comment by Yoan-MA** [comment]
sure, go ahead ! Maybe we could put the word in italic to indicate that it's a french word ?

All my suggestions are not about `arrondissement` and `districts`. Please take a look on the others 🙏

File: README.md [hunk 5]
```diff
@@ -1,53 +1,53 @@
...
+A documentation detailing how the project is designed can be found [here](./usages.md) to understand how to start the project, interact and enter into containers if needed, ...
 
-## 1 - Collecter l’information
+## 1 - Collecting data
 
-On souhaite collecter l'ensemble des annonces de biens immobilier sur Paris. Ces informations seront à récupérer depuis l'API d'annonces fournie (`listingapi`), qu'il faudra parcourir entièrement.
+We want to collect the entirety of real estate properties in Paris. These properties will be retrieved from the real estate properties API (`listingapi`), that you will need to browse entirely.
 
-**Il ne faut donc pas modifier `listingapi`, l'API ne sert qu'à servir les données.**
+**You must not change the `listingapi`, it only serves at exposing data.**
```
Comment (corenting) [inline README.md:31]:
```suggestion
**You must not change the `listingapi`, it is only used to expose data.**
```

File: README.md [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
-- depuis le conteneur de votre application `pricemap` via: http://listingapi:5000/listings/32682
+You can access this API, after starting the project :
+- from your local environment, at: http://localhost:8181/listings/32682
+- from the container of your `pricemap` application at: http://listingapi:5000/listings/32682
 
-Afin de commencer, un premier jet du code existe déjà dans le fichier `app.py`. Ce code ajoute un endpoint `/update_data` à l'application `pricemap` qui lance la récupération de la donnée.
-**Le code est fonctionnel, mais il existe plusieurs améliorations possibles.**
-L'objectif de cette partie est de retravailler et compléter ce code déjà existant, afin de le rendre plus propre et plus proche de l'état de l'art.
-Cette partie est libre, il est donc possible de déplacer le code, d'ajouter des bibliothèques, de changer la façon dont le code est appelé, etc.
+To get started, a first implementation already exists in the `app.py` file. This code adds a `/update_data` endpoint to the `pricemap` app, which starts data retrieval.
```
Comment (corenting) [inline README.md:37]: To linked to the translation, but:
```suggestion
To get started, a first implementation already exists in the `app.py` file. This code adds a `/update_data` endpoint to the `pricemap` app, which starts the data retrieval process. If you feels it is necessary to change the way the data retrevial process is started, don't hesitate to change it.
```

File: usages.md [hunk 7]
```diff
@@ -1,38 +1,38 @@
...
 make run
 ```
 
-`ctrl + C` pour les stopper
+`ctrl + C` to stop.
 
-## Reconstruire le projet
-Normalement cette étape n'est pas nécessaire car effectuée automatiquement au démarrage des applications mais permet de forcer la reconstruction des applications.
+## Rebuilding the project
+This should not be necessary as done automatically on first start, but can allow to force the rebuilt of the apps.
```
Comment (corenting) [inline usages.md:13]:
```suggestion
This should not be necessary as done automatically on first start, but can allow to force the rebuild of the apps.
```

File: usages.md [hunk 8]
```diff
@@ -1,38 +1,38 @@
...
 docker-compose run pricemap bash
 ```
 
-Vous pouvez également le faire sur `listingapi` et `db` mais cela ne devrait pas être utile.
+You can also do it on the `listingapi` and `db`, but it should not be necessary.
 
-## Nettoyage
-Détruit les conteneurs des applications. Utile car par défaut, `docker-compose` réutilise des conteneurs déjà lancés
+## Cleaning-up
+Destroys application containers. Useful as `docker-compose` reuses already existing containers by default.
```
Comment (corenting) [inline usages.md:29]:
```suggestion
Destroy application containers. Useful as `docker-compose` reuses already existing containers by default.
```

File: usages.md [hunk 9]
```diff
@@ -1,38 +1,38 @@
...
+This should not be necessary as done automatically on first start, but can allow to force the rebuilt of the apps.
 
 ```sh
 make build
 ```
 
-## Entrer dans le conteneur
-Pour entrer dans le shell de l'application et taper des commandes manuelles
+## Entering a container
+To inter in the shell of the app and run manual commands.
```
Comment (corenting) [inline usages.md:20]:
```suggestion
To enter the shell of the app container and run manual commands.
```

**Review by corenting — CHANGES_REQUESTED** [review]

**Review by Yoan-MA — APPROVED** [review]
🚀

**Pull request merged**